presentation.md

```diff
@@ -94,25 +104,45 @@ cała aglomeracja przewyższa populacją Warszawę.
 
 # Jak pracowaliśmy z danymi
 
+::: notes
 Te dane musiały zostac sprowadzone do wspólnego formatu, tak aby wszystkie
 stworzone wykresy i analizy można było odtwarzać natychmiast po aktualizacji
 danych, ponieważ te zmieniały się z dnia na dzień wraz z rozwojem pandemii. Do
 tego wykorzystaliśmy pythona i Jupyterlab.
+::: 
 
 # Jak pracowaliśmy z danymi
 
+::: notes
 Z pomoca jupyterlab można stworzyć interaktywne notatniki, które działają krok
 po kroku, jeden z nich służył wyłącznie do pobrania danych ze wszystkich źródeł
 ich oczyszczenia i sprowadzenia do wspólnego formatu. Same analizy znalazły się
 w osobnym notatniku, co umożliwiło nam zapanowanie nad projektem. Oczywiście
 korzystanie z notatników też ma swoje wady, kolejne zmiany utrzymywane w
 systemie kontroli wersji Git nie są czytalne.
+:::
 
 ![Wykres zarażeń województwa + Polska](img/zar.png)
 
-
 # Wnioski
 
-## Czego się nauczyliśmy z tej analizy?
-
-## Co innego zyskaliśmy?
\ No newline at end of file
+## Czego się nauczyliśmy?
+
+::: notes
+Analiza tych danych pozwoliła nam na zapoznanie się z metodami przetwarzznia ich,
+podstawami wizualizacji, a także znajdowania nowych źródeł. Bardzo dużą rolę
+odegrało także oczyszczanie tych danych, ponieważ prawie wszystkie źródła
+zawierały dane niepotrzebne, które w żaden sposób nie przyczyniają się do
+jakichkolwiek znaczących wyników. Wykluczyliśmy także dane, które silnie
+korelowały ze sobą, ponieważ mogły one zakłócić odbiór pozostałych statystyk.
+:::
+
+## Co innego zyskaliśmy?
+
+::: notes
+Nasze _badania_ przełożyły się poza tym wszystkim na dodatkowe punkty do
+stypendium rektora, ponieważ były powiązane z innymi badaniami prowadzonymi na
+PRz przez prof. Dominika Strzałkę. Dzięki temu mogliśmy także lepiej poznać się
+z kolegami z roku, mimo sytuacji pandemicznej, ponieważ wymienialiśmy się
+doświadczeniami i rozmawialiśmy na temat całego hackathonu.
+:::
```

Comment:
Warto by dopisać o tym, że to doświadczenie bardzo różni się od projektów uczelnianych. Tutaj nikt nie daje nam jasnego celu, danych, a od mentorów mieliśmy wyłącznie pomoc merytoryczną czy wskazówki, czy dane rozwiązanie może się sprawdzić.